contracts/contracts/gatekeepers/EASGatekeeper.sol

```diff
@@ -11,7 +11,7 @@ import { IEAS } from "../interfaces/IEAS.sol";
 /// only if they've received an attestation of a specific schema from a trusted attester
 contract EASGatekeeper is SignUpGatekeeper, Ownable(msg.sender) {
   // the reference to the EAS contract
-  IEAS private immutable eas;
+  IEAS public immutable eas;
```

Comment:
why we make this private before, and public now?

Reply:
not sure why it was private, maybe to not allow to auto generate the public view function eas() - but I saw the hats gatekeeper had them public so thought why not keep them all public. If you remember once we deployed with this value wrong and had issues troubleshooting, now it should be easier 🤞